integration/etcd.go

```diff
@@ -63,7 +72,11 @@ func (e *Etcd) Start() error {
 
 	e.processState.StartMessage = fmt.Sprintf("serving insecure client requests on %s", e.processState.URL.Hostname())
 
-	return e.processState.Start()
+	e.processState.CopyTo(
```

Comment:
Just setting the values that you want should be good (since you have to remember the order of the arguments etc anyway).